js/src/builder/float.ts

```diff
@@ -21,8 +21,30 @@ import { Float, Float16, Float32, Float64 } from '../type';
 /** @ignore */
 export class FloatBuilder<T extends Float = Float, TNull = any> extends FixedWidthBuilder<T, TNull> {}
 
+const minFloat16 = -1;
+const maxFloat16 = (65535 - 32767) / 32767;
```

Comment:
Where do these numbers come from? Can you add a comment?

Reply:
Maybe extract all the magic numbers into constants. Like `32767`, `32767`, and `65536`.

Reply:
This is the magic formula to create a float16 from a uint32 (65535 is max half float aka 2^16-1, 32767 is 2^15-1). From what I understand of how this works, subtracting 2^15 from the target sets the lowest 15 bits to 0, then dividing by 2^15 is essentially a right shift 15 to mask off the remaining bits. We use this formula in other places like the get and set routines for the Float16 types as well.

Reply:
Would it make sense to add constants such as `const halfFloat = 2^16-1;` then?

Reply:
+1 - we have a number of places with magic numbers and computed constants like this that could be better documented, maybe in a single constants file?